docs/source/conf.py

```diff
@@ -346,7 +346,10 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = []
+exclude_patterns = [
+    # Temporarily hide certain docs until they are finalized
+    'distributed_shard.rst'
```

Comment:
What is this doing? If the corresponding functions are public, then the website must document them.